recipe/meta.yaml

```diff
@@ -23,11 +23,11 @@ requirements:
     - bc
     - swig >=3.0.7
   host:
-    - blas 1.1 openblas
-    - openblas
+    - mkl-devel {{ mkl }}  # [blas_impl == "mkl"]
+    - openblas-devel {{ openblas }}  # [blas_impl != "mkl"]
```

Comment:
Don't use `openblas-devel`. It's from defaults and incompatible with conda-forge

Reply:
@isuruf, what should we use instead?

Reply:
BTW, should we split our `opeblas` package too?

Reply:
I have switched to using simply `openblas`, is that right?

Reply:
@isuruf, can you take another quick look at this please, thanks?

Reply:
@duncanmmacleod I think you may have force pushed `-devel` back on your fork+branch.